### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by phil-davis**
Mostly in comments, but some variable and function names. IMO it is all harmless and does not either fix or break anything.

### CODE REVIEW

File: hugolib/hugo_modules_test.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ JS imported in module: |
         "devDependencies": {
                 "postcss-cli": "7.8.0",
                 "tailwindcss": "1.8.0"
- 
+
```
Comment (moorereason) [inline hugolib/hugo_modules_test.go:154]: Why the whitespace?

File: hugolib/hugo_modules_test.go [hunk 2]
```diff
@@ -1078,7 +1078,7 @@ workingDir=%q
   [[module.mounts]]
     source = %q
     target = "content"
-    
+
```
Comment (moorereason) [inline hugolib/hugo_modules_test.go:1081]: Why the whitespace?

File: hugolib/pagebundler_test.go [hunk 3]
```diff
@@ -1250,7 +1250,7 @@ func TestBundleTransformMany(t *testing.T) {
 title: "Page"
 weight: %d
 ---
-		
+
```
Comment (moorereason) [inline hugolib/pagebundler_test.go:1253]: Why the whitespace?

File: hugolib/taxonomy_test.go [hunk 4]
```diff
@@ -316,7 +316,7 @@ categories: ["This is Cool", "And new" ]
 ---
 
 Content.
-		
+
```
Comment (moorereason) [inline hugolib/taxonomy_test.go:319]: Why the whitespace?

File: hugolib/taxonomy_test.go [hunk 5]
```diff
@@ -525,7 +525,7 @@ Funny:|/p2/|`)
 func TestTaxonomiesListPages(t *testing.T) {
 	b := newTestSitesBuilder(t)
 	b.WithTemplates("_default/list.html", `
-	
+
```
Comment (moorereason) [inline hugolib/taxonomy_test.go:528]: Why the whitespace?

File: hugolib/taxonomy_test.go [hunk 6]
```diff
@@ -560,7 +560,7 @@ categories: ["cats"]
 	b.Build(BuildCfg{})
 
 	b.AssertFileContent("public/index.html", `
-    
+
```
Comment (moorereason) [inline hugolib/taxonomy_test.go:563]: Why the whitespace?

File: resources/page/permalinks_test.go [hunk 7]
```diff
@@ -38,7 +38,7 @@ var testdataPermalinks = []struct {
 	{"/:filename/", true, "/test-page/"},                            // Filename
 	{"/:06-:1-:2-:Monday", true, "/12-4-6-Friday"},                  // Dates with Go formatting
 	{"/:2006_01_02_15_04_05.000", true, "/2012_04_06_03_01_59.000"}, // Complicated custom date format
-	// TODO(moorereason): need test scaffolding for this.
+	// TODO(more reason): need test scaffolding for this.
```
Comment (moorereason) [inline resources/page/permalinks_test.go:41]: Not this.  `moorereason` is my GH username.

**Review by moorereason — CHANGES_REQUESTED** [review]
Most of this is correct and helpful.  Thanks for taking the time to clean this up.  However...

One edit is wrong (my username).  Others add whitespace for no apparent reason.  What's the rationale for the extra whitespace?

File: hugolib/hugo_modules_test.go (hunk 1 above)
Comment (moorereason) [reply hugolib/hugo_modules_test.go:154]: Nevermind.  I see the empty space now that's I've commented.

Comment (phil-davis) [reply hugolib/hugo_modules_test.go:154]: My IDE auto-removed whitespace at end-of-line in files that I touched.

File: resources/page/permalinks_test.go (hunk 7 above)
Comment (phil-davis) [reply resources/page/permalinks_test.go:41]: reverted

**Review by moorereason — APPROVED** [review]
👍

**Pull request merged**